lib/src/components/NxList/__tests__/NxListActions.test.tsx

```diff
@@ -4,24 +4,50 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+import React from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
 import NxList from '../NxList';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
+import NxButton from '../../NxButton/NxButton';
+
+describe('NxList.Actions', function() {
+  const quickRender = rtlRender(NxList.Actions, {}),
+      renderEl = rtlRenderElement(NxList.Actions, {});
+
+  it('sets the specified classnames in addition to the defaults', function() {
+    const el = renderEl({ className: 'foo' }),
+        defaultEl = renderEl()!;
 
-describe('NxListActions', function() {
+    expect(el).toHaveClass('foo');
+
+    for (const cls of Array.from(defaultEl.classList)) {
+      expect(el).toHaveClass(cls);
+    }
+  });
 
-  const minimalProps = {};
-  const getShallow = getShallowComponent(NxList.Actions, minimalProps);
+  it('sets the specified attrs', function() {
+    const el = renderEl({ id: 'foo', lang: 'en' });
 
-  it('renders a div with class nx-list__actions', function() {
-    const contentEl = getShallow();
-    expect(contentEl).toExist();
-    expect(contentEl.find('div')).toHaveClassName('.nx-list__actions');
+    expect(el).toHaveAttribute('id', 'foo');
+    expect(el).toHaveAttribute('lang', 'en');
   });
 
-  it('renders the classNames given to it', function() {
-    const contentEl = getShallow({className: 'customClassName'});
-    expect(contentEl).toExist();
-    expect(contentEl.find('div')).toMatchSelector('.nx-list__actions.customClassName');
+  it('sets a ref to the element', function() {
+    const ref = React.createRef<HTMLDivElement>(),
+        el = renderEl({ ref });
+
+    expect(ref.current).toBe(el);
+  });
+
+  it('renders children correctly', function() {
+    const children = <NxButton>Foo</NxButton>,
+        view = quickRender({ children }),
+        el = view.container.querySelector('div.nx-list__actions'),
```

Comment:
`el` is just the top-level element right? Use `view.container.firstElementChild` instead to avoid impl-detail classnames

Reply:
Yes. Updated here f31d3f4a01144fd2931367a77813b963191affe1